x-pack/plugins/enterprise_search/server/__mocks__/routerDependencies.mock.ts

```diff
@@ -5,7 +5,7 @@
  */
```

Comment:
Just curious, why were these file moved out of `routes/__mocks__`? Since it's a router mock, doesn't it still makes sense to keep it in `routes/`?

Reply:
I should add that I don't feel super strongly about it and actually thought about moving it myself at one point in the past, would just love to hear someone else's thinking 😄

Reply:
Basically, I needed the mockLogger and mockConfig for the `enterprise_search_request_handler.test.js` tests. I felt weird reaching *down* the tree into the `routes` directory to get those. So since it seemed like a number of packages within `/server` could possibly need the same mocks, I simply moved the directory up. For `routerDependencies.mock.ts` that would necessarily have been required, but I thought I'd just move everything together. What do you suggest I do at this point? Keep it or change it? I'm happy to do either.

Reply:
I'm kinda 50/50, I don't think it's all that weird to reach down into `routes/` to grab the mock personally since it's a route helper lib 🤷 I don't super mind either way, feel free to leave it!